java/dagger/example/android/espresso/main/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.example.android.espresso">
+
+  <uses-sdk
+    android:minSdkVersion="14"
+    android:targetSdkVersion="24"/>
+
+  <application
+    android:label="Espresso Dagger"
```

Comment:
I'm open to that. Depends on where folks want to go with this sample. 

I imagined that we might eventually want to demo best practices for solving some espresso-specific problems with Dagger. For example, we might want to show how to setup modules and components so that you can inject dependencies that implement `IdlingResource` for a test build variant. Maybe that's a bad example, but if we want to show any espresso specific stuff, then maybe the name should stay.